Add build-environment mojo

frauzufall opened the pull request:
Goal: create a mojo to build a scijava based app from scratch (e.g. by running the mojo in the `fiji/fiji` repository)

frauzufall commented:
I just copied the copy-jars mojo so far. Built the app by running `mvn -Dscijava.app.directory=/PATH/TO/Fiji.build org.scijava:scijava-maven-plugin:1.1.2-SNAPSHOT:build-environment`. Here's the comparison to the currently downloadable Fiji (linux64, `Fiji.app` is the downloaded one, `Fiji.build` the one built from the mojo):
```
(base) ➜  build-environment diff -r Fiji.build Fiji.app 
Only in Fiji.app: Contents
Only in Fiji.app: db.xml.gz
Only in Fiji.app: ImageJ-linux64
Only in Fiji.app: images
Only in Fiji.app/jars: asm-analysis-4.0.jar
Only in Fiji.app/jars: autocomplete-2.6.1.jar
Only in Fiji.build/jars: autocomplete-3.0.0.jar
Only in Fiji.app/jars: batik-anim-1.10.jar
Only in Fiji.build/jars: batik-anim-1.11.jar
Only in Fiji.app/jars: batik-awt-util-1.10.jar
Only in Fiji.build/jars: batik-awt-util-1.11.jar
Only in Fiji.app/jars: batik-bridge-1.10.jar
Only in Fiji.build/jars: batik-bridge-1.11.jar
Only in Fiji.app/jars: batik-constants-1.10.jar
Only in Fiji.build/jars: batik-constants-1.11.jar
Only in Fiji.app/jars: batik-css-1.10.jar
Only in Fiji.build/jars: batik-css-1.11.jar
Only in Fiji.app/jars: batik-dom-1.10.jar
Only in Fiji.build/jars: batik-dom-1.11.jar
Only in Fiji.app/jars: batik-ext-1.10.jar
Only in Fiji.build/jars: batik-ext-1.11.jar
Only in Fiji.app/jars: batik-gvt-1.10.jar
Only in Fiji.build/jars: batik-gvt-1.11.jar
Only in Fiji.app/jars: batik-i18n-1.10.jar
Only in Fiji.build/jars: batik-i18n-1.11.jar
Only in Fiji.app/jars: batik-parser-1.10.jar
Only in Fiji.build/jars: batik-parser-1.11.jar
Only in Fiji.app/jars: batik-script-1.10.jar
Only in Fiji.build/jars: batik-script-1.11.jar
Only in Fiji.app/jars: batik-svg-dom-1.10.jar
Only in Fiji.build/jars: batik-svg-dom-1.11.jar
Only in Fiji.app/jars: batik-svggen-1.10.jar
Only in Fiji.build/jars: batik-svggen-1.11.jar
Only in Fiji.app/jars: batik-util-1.10.jar
Only in Fiji.build/jars: batik-util-1.11.jar
Only in Fiji.app/jars: batik-xml-1.10.jar
Only in Fiji.build/jars: batik-xml-1.11.jar
Only in Fiji.build/jars: bigdataviewer_fiji-6.0.0.jar
Only in Fiji.build/jars/bio-formats: formats-api-6.3.0.jar
Only in Fiji.app/jars/bio-formats: formats-api-6.4.0.jar
Only in Fiji.build/jars/bio-formats: formats-bsd-6.3.0.jar
Only in Fiji.app/jars/bio-formats: formats-bsd-6.4.0.jar
Only in Fiji.build/jars/bio-formats: formats-gpl-6.3.0.jar
Only in Fiji.app/jars/bio-formats: formats-gpl-6.4.0.jar
Only in Fiji.app/jars/bio-formats: jai_imageio-5.1.10.jar
Only in Fiji.build/jars/bio-formats: jxrlib-all-0.2.1.jar
Only in Fiji.app/jars/bio-formats: jxrlib-all-0.2.2.jar
Only in Fiji.app/jars/bio-formats: metakit-5.3.2.jar
Only in Fiji.app/jars/bio-formats: ome-codecs-0.3.0.jar
Only in Fiji.app/jars/bio-formats: ome-common-6.0.4.jar
Only in Fiji.app/jars/bio-formats: ome-jai-0.1.0.jar
Only in Fiji.app/jars/bio-formats: ome-mdbtools-5.3.2.jar
Only in Fiji.app/jars/bio-formats: ome-poi-5.3.3.jar
Only in Fiji.app/jars/bio-formats: ome-xml-6.0.1.jar
Only in Fiji.app/jars/bio-formats: specification-6.0.1.jar
Only in Fiji.build/jars/bio-formats: turbojpeg-6.3.0.jar
Only in Fiji.app/jars/bio-formats: turbojpeg-6.4.0.jar
Only in Fiji.build/jars: c3p0-0.9.5.3.jar
Only in Fiji.build/jars: cdm-4.6.13.jar
Only in Fiji.app/jars: commons-codec-1.11.jar
Only in Fiji.build/jars: commons-codec-1.12.jar
Only in Fiji.build/jars: commons-compress-1.18.jar
Only in Fiji.app/jars: commons-compress-1.4.1.jar
Only in Fiji.app/jars: commons-httpclient-3.1.jar
Only in Fiji.app/jars: commons-lang3-3.8.1.jar
Only in Fiji.build/jars: commons-lang3-3.9.jar
Only in Fiji.build/jars: commons-text-1.6.jar
Only in Fiji.build/jars: Correct_3D_Drift-1.0.3.jar
Only in Fiji.build/jars: ffmpeg-4.1-1.4.4-android-arm64.jar
Only in Fiji.build/jars: ffmpeg-4.1-1.4.4-android-arm.jar
Only in Fiji.build/jars: ffmpeg-4.1-1.4.4-android-x86_64.jar
Only in Fiji.build/jars: ffmpeg-4.1-1.4.4-android-x86.jar
Only in Fiji.build/jars: ffmpeg-4.1-1.4.4-linux-armhf.jar
Only in Fiji.build/jars: ffmpeg-4.1-1.4.4-linux-ppc64le.jar
Only in Fiji.build/jars: ffmpeg-4.1-1.4.4-linux-x86_64.jar
Only in Fiji.build/jars: ffmpeg-4.1-1.4.4-linux-x86.jar
Only in Fiji.build/jars: ffmpeg-4.1-1.4.4-macosx-x86_64.jar
Only in Fiji.build/jars: ffmpeg-4.1-1.4.4-windows-x86_64.jar
Only in Fiji.build/jars: ffmpeg-4.1-1.4.4-windows-x86.jar
Only in Fiji.app/jars: fiji-2.0.0-pre-8.jar
Only in Fiji.build/jars: fiji-2.0.0-SNAPSHOT.jar
Only in Fiji.app/jars: fiji-compat-2.0.3.jar
Only in Fiji.app/jars: gentyref-1.1.0.jar
Only in Fiji.app/jars: gluegen-rt-2.3.2.jar
Only in Fiji.build/jars: gluegen-rt-2.3.2-natives-android-aarch64.jar
Only in Fiji.build/jars: google-http-client-1.20.0.jar
Only in Fiji.build/jars: google-http-client-xml-1.20.0.jar
Only in Fiji.app/jars: groovy-json-2.4.8.jar
Only in Fiji.app/jars: groovy-templates-2.4.8.jar
Only in Fiji.app/jars: groovy-xml-2.4.8.jar
Only in Fiji.build/jars: httpclient-4.5.1.jar
Only in Fiji.app/jars: httpclient-4.5.6.jar
Only in Fiji.app/jars: httpcore-4.4.11.jar
Only in Fiji.build/jars: httpmime-4.5.1.jar
Only in Fiji.build/jars: httpservices-4.6.13.jar
Only in Fiji.app/jars: ij-1.52p.jar
Only in Fiji.build/jars: ij-1.52r.jar
Only in Fiji.app/jars: ij1-patcher-0.12.9.jar
Only in Fiji.build/jars: ij1-patcher-1.0.0.jar
Only in Fiji.app/jars: imagej-2.0.0-rc-69.jar
Only in Fiji.build/jars: imagej-2.0.0-rc-72.jar
Only in Fiji.app/jars: imagej-common-0.28.2.jar
Only in Fiji.build/jars: imagej-common-0.31.0.jar
Only in Fiji.app/jars: imagej-launcher-4.0.5.jar
Only in Fiji.build/jars: imagej-legacy-0.35.0.jar
Only in Fiji.app/jars: imagej-legacy-0.35.2.jar
Only in Fiji.app/jars: imagej-mesh-io-0.1.2.jar
Only in Fiji.app/jars: imagej-notebook-0.3.0.jar
Only in Fiji.build/jars: imagej-notebook-0.7.1.jar
Only in Fiji.app/jars: imagej-ops-0.43.1.jar
Only in Fiji.build/jars: imagej-ops-0.45.2.jar
Only in Fiji.build/jars: imagej-plugins-uploader-webdav-0.2.2.jar
Only in Fiji.app/jars: imagej-plugins-uploader-webdav-0.3.1.jar
Only in Fiji.build/jars: imagej-ui-swing-0.22.0.jar
Only in Fiji.app/jars: imagej-ui-swing-0.23.1.jar
Only in Fiji.app/jars: imagej-updater-0.10.4.jar
Only in Fiji.build/jars: imagej-updater-0.9.3.jar
Only in Fiji.app/jars: imglib2-5.6.3.jar
Only in Fiji.build/jars: imglib2-5.8.0.jar
Only in Fiji.app/jars: imglib2-roi-0.7.0.jar
Only in Fiji.build/jars: imglib2-roi-0.8.1.jar
Only in Fiji.build/jars: itextpdf-5.5.13.1.jar
Only in Fiji.app/jars: itextpdf-5.5.13.jar
Only in Fiji.app/jars: jackrabbit-webdav-2.19.1.jar
Only in Fiji.build/jars: jackson-annotations-2.9.8.jar
Only in Fiji.build/jars: jackson-core-2.9.8.jar
Only in Fiji.build/jars: jackson-databind-2.9.8.jar
Only in Fiji.app/jars: javassist-3.24.0-GA.jar
Only in Fiji.build/jars: javassist-3.25.0-GA.jar
Only in Fiji.app/jars: jcommon-1.0.24.jar
Binary files Fiji.build/jars/jep-2.4.2.jar and Fiji.app/jars/jep-2.4.2.jar differ
Only in Fiji.build/jars: jfreesvg-3.4.jar
Only in Fiji.app/jars: jgrapht-0.0.0-STUB.jar
Only in Fiji.app/jars: jgrapht-core-1.3.0.jar
Only in Fiji.build/jars: jgrapht-core-1.3.1.jar
Only in Fiji.build/jars: jheaps-0.10.jar
Only in Fiji.app/jars: jheaps-0.9.jar
Only in Fiji.app/jars: jitk-tps-3.0.0.jar
Only in Fiji.build/jars: jitk-tps-3.0.1.jar
Only in Fiji.app/jars: jnr-ffi-1.0.7.jar
Only in Fiji.app/jars: joal-2.3.2.jar
Only in Fiji.build/jars: joal-2.3.2-natives-android-aarch64.jar
Only in Fiji.app/jars: jocl-2.3.2.jar
Only in Fiji.build/jars: jocl-2.3.2-natives-android-aarch64.jar
Only in Fiji.app/jars: joda-time-2.10.1.jar
Only in Fiji.build/jars: joda-time-2.10.2.jar
Only in Fiji.app/jars: jogl-all-2.3.2.jar
Only in Fiji.build/jars: jogl-all-2.3.2-natives-android-aarch64.jar
Only in Fiji.app/jars: joml-1.9.12.jar
Only in Fiji.build/jars: joml-1.9.16.jar
Only in Fiji.app/jars: jply-0.2.0.jar
Only in Fiji.build/jars: jython-shaded-2.7.1.1.jar
Only in Fiji.app/jars: jython-shaded-2.7.1.jar
Only in Fiji.build/jars: KymographBuilder-2.1.1.jar
Only in Fiji.app/jars: languagesupport-2.6.0.jar
Only in Fiji.build/jars: languagesupport-3.0.0.jar
Only in Fiji.app/jars: linux64
Only in Fiji.app/jars: mapdb-1.0.3.jar
Only in Fiji.build/jars: mchange-commons-java-0.2.15.jar
Only in Fiji.build/jars: metadata-extractor-2.11.0.jar
Only in Fiji.app/jars: metadata-extractor-2.6.2.jar
Only in Fiji.build/jars: metakit-5.3.2.jar
Only in Fiji.app/jars: minimaven-2.2.1.jar
Only in Fiji.build/jars: minimaven-2.2.2.jar
Only in Fiji.build/jars: minio-5.0.2.jar
Only in Fiji.app/jars: native-lib-loader-2.3.2.jar
Only in Fiji.build/jars: native-lib-loader-2.3.4.jar
Only in Fiji.app/jars: netcdf-4.3.22.jar
Only in Fiji.build/jars: okhttp-3.6.0.jar
Only in Fiji.build/jars: okio-1.11.0.jar
Only in Fiji.build/jars: ome-codecs-0.2.5.jar
Only in Fiji.build/jars: ome-common-6.0.4.jar
Only in Fiji.build/jars: ome-jai-0.1.0.jar
Only in Fiji.build/jars: ome-mdbtools-5.3.2.jar
Only in Fiji.build/jars: ome-poi-5.3.3.jar
Only in Fiji.build/jars: ome-xml-6.0.1.jar
Only in Fiji.app/jars: perf4j-0.9.13.jar
Only in Fiji.build/jars: perf4j-0.9.16.jar
Only in Fiji.app/jars: rsyntaxtextarea-2.6.1.jar
Only in Fiji.build/jars: rsyntaxtextarea-3.0.3.jar
Only in Fiji.app/jars: scijava-common-2.80.1.jar
Only in Fiji.build/jars: scijava-common-2.82.0.jar
Only in Fiji.build/jars: scijava-io-http-0.2.0.jar
Only in Fiji.build/jars: scijava-plugins-io-table-0.2.0.jar
Only in Fiji.app/jars: scijava-search-0.5.0.jar
Only in Fiji.build/jars: scijava-search-0.6.0.jar
Only in Fiji.app/jars: scijava-table-0.2.0.jar
Only in Fiji.build/jars: scijava-table-0.4.0.jar
Only in Fiji.app/jars: scijava-ui-swing-0.12.0.jar
Only in Fiji.build/jars: scijava-ui-swing-0.13.0.jar
Only in Fiji.build/jars: script-editor-0.5.0.jar
Only in Fiji.app/jars: script-editor-0.5.1.jar
Only in Fiji.app/jars: serializer-2.7.2.jar
Only in Fiji.app/jars: slf4j-api-1.7.25.jar
Only in Fiji.build/jars: slf4j-api-1.7.26.jar
Only in Fiji.app/jars: snakeyaml-1.23.jar
Only in Fiji.build/jars: snakeyaml-1.24.jar
Only in Fiji.build/jars: specification-6.0.1.jar
Only in Fiji.app/jars: T2-NIT-1.1.2.jar
Only in Fiji.build/jars: T2-NIT-1.1.3.jar
Only in Fiji.app/jars: T2-TreelineGraph-1.1.2.jar
Only in Fiji.build/jars: T2-TreelineGraph-1.1.3.jar
Only in Fiji.build/jars: VectorGraphics2D-0.13.jar
Only in Fiji.app/jars: xalan-2.7.2.jar
Only in Fiji.build/jars: xchart-3.5.4.jar
Only in Fiji.app/jars: xmpcore-5.1.2.jar
Only in Fiji.build/jars: xmpcore-5.1.3.jar
Only in Fiji.build/jars: xpp3-1.1.4c.jar
Only in Fiji.app/jars: xz-1.0.jar
Only in Fiji.app/jars: yecht-1.0.jar
Only in Fiji.app: java
Only in Fiji.app: lib
Only in Fiji.app: luts
Only in Fiji.app: macros
Only in Fiji.app/plugins: Analyze
Only in Fiji.app/plugins: AnalyzeSkeleton_-3.3.0.jar
Only in Fiji.build/plugins: AnalyzeSkeleton_-3.4.1.jar
Only in Fiji.app/plugins: bigdataviewer_fiji-6.0.0.jar
Only in Fiji.build/plugins: bio-formats_plugins-6.3.0.jar
Only in Fiji.app/plugins: bio-formats_plugins-6.4.0.jar
Only in Fiji.build/plugins: bUnwarpJ_-2.6.11.jar
Only in Fiji.app/plugins: bUnwarpJ_-2.6.12.jar
Only in Fiji.app/plugins: Correct_3D_Drift-1.0.3.jar
Only in Fiji.app/plugins: Examples
Only in Fiji.app/plugins: JRuby
Only in Fiji.app/plugins: KymographBuilder-2.1.1.jar
Only in Fiji.app/plugins: Macros
Only in Fiji.app/plugins: readme.txt
Only in Fiji.build/plugins: register_virtual_stack_slices-3.0.5.jar
Only in Fiji.app/plugins: register_virtual_stack_slices-3.0.7.jar
Only in Fiji.app/plugins: Scripts
Only in Fiji.build/plugins: Stack_Manipulation-2.1.1.jar
Only in Fiji.app/plugins: Stack_Manipulation-2.1.2.jar
Only in Fiji.build/plugins: TrackMate_-4.0.0.jar
Only in Fiji.app/plugins: TrackMate_-5.2.0.jar
Only in Fiji.build/plugins: Trainable_Segmentation-3.2.29.jar
Only in Fiji.app/plugins: Trainable_Segmentation-3.2.34.jar
Only in Fiji.app/plugins: Utilities
Only in Fiji.app/plugins: Vaa3d_Writer-1.0.2.jar
Only in Fiji.build/plugins: Vaa3d_Writer-1.0.3.jar
Only in Fiji.app: README.md
Only in Fiji.app: retro
Only in Fiji.app: scripts
Only in Fiji.app: WELCOME.md

```

ctrueden commented:
@frauzufall Cool!

I think it would help a lot to do a color-words diff, rather than a normal line-by-line diff, because a lot of those differences are just version numbers. Those don't illustrate gaps in the mojo, but things like missing scripts and other resources, as well as JARs in the wrong place, do.

frauzufall commented:
Yes.. I tried a couple of command line comparison tools but nothing was super helpful or colored.. I wrote a helper class a while ago for the maven updater prototype, it prints this (not bulletproof, but helpful):
<details><summary>file changes via CompareDirectories.java</summary>
<p>

```ssh
Comparing /home/random/Development/imagej/project/updater/build-environment/Fiji.build with /home/random/Development/imagej/project/updater/build-environment/Fiji.app:
ADDED /Contents/Info.plist
ADDED /Contents/Resources/Fiji.icns
ADDED /Contents/Resources/ImageJ.icns
ADDED /ImageJ-linux64
ADDED /README.md
ADDED /WELCOME.md
ADDED /db.xml.gz
ADDED /images/about/about1.tif
ADDED /images/about/about1.tif.txt
ADDED /images/about/about2.tif
ADDED /images/about/about2.tif.txt
ADDED /images/about/about3.tif
ADDED /images/about/about3.tif.txt
ADDED /images/about/about4.tif
ADDED /images/about/about4.tif.txt
ADDED /images/about/about5.tif
ADDED /images/about/about5.tif.txt
ADDED /images/icon-flat.png
ADDED /images/icon.png
ADDED /jars/asm-analysis-4.0.jar
ADDED /jars/bio-formats/jai_imageio-5.1.10.jar
ADDED /jars/bio-formats/ome-codecs-0.3.0.jar
ADDED /jars/commons-httpclient-3.1.jar
ADDED /jars/fiji-compat-2.0.3.jar
ADDED /jars/gentyref-1.1.0.jar
ADDED /jars/groovy-json-2.4.8.jar
ADDED /jars/groovy-templates-2.4.8.jar
ADDED /jars/groovy-xml-2.4.8.jar
ADDED /jars/httpcore-4.4.11.jar
ADDED /jars/imagej-launcher-4.0.5.jar
ADDED /jars/imagej-mesh-io-0.1.2.jar
ADDED /jars/jackrabbit-webdav-2.19.1.jar
ADDED /jars/jcommon-1.0.24.jar
ADDED /jars/jgrapht-0.0.0-STUB.jar
ADDED /jars/jnr-ffi-1.0.7.jar
ADDED /jars/jply-0.2.0.jar
ADDED /jars/linux64/gluegen-rt-2.3.2-natives-linux-amd64.jar
ADDED /jars/linux64/joal-2.3.2-natives-linux-amd64.jar
ADDED /jars/linux64/jocl-2.3.2-natives-linux-amd64.jar
ADDED /jars/linux64/jogl-all-2.3.2-natives-linux-amd64.jar
ADDED /jars/mapdb-1.0.3.jar
ADDED /jars/netcdf-4.3.22.jar
ADDED /jars/serializer-2.7.2.jar
ADDED /jars/xalan-2.7.2.jar
ADDED /jars/xz-1.0.jar
ADDED /jars/yecht-1.0.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/COPYRIGHT
ADDED /java/linux-amd64/jdk1.8.0_172/jre/LICENSE
ADDED /java/linux-amd64/jdk1.8.0_172/jre/README
ADDED /java/linux-amd64/jdk1.8.0_172/jre/THIRDPARTYLICENSEREADME-JAVAFX.txt
ADDED /java/linux-amd64/jdk1.8.0_172/jre/THIRDPARTYLICENSEREADME.txt
ADDED /java/linux-amd64/jdk1.8.0_172/jre/Welcome.html
ADDED /java/linux-amd64/jdk1.8.0_172/jre/bin/ControlPanel
ADDED /java/linux-amd64/jdk1.8.0_172/jre/bin/java
ADDED /java/linux-amd64/jdk1.8.0_172/jre/bin/javaws
ADDED /java/linux-amd64/jdk1.8.0_172/jre/bin/jcontrol
ADDED /java/linux-amd64/jdk1.8.0_172/jre/bin/jjs
ADDED /java/linux-amd64/jdk1.8.0_172/jre/bin/keytool
ADDED /java/linux-amd64/jdk1.8.0_172/jre/bin/orbd
ADDED /java/linux-amd64/jdk1.8.0_172/jre/bin/pack200
ADDED /java/linux-amd64/jdk1.8.0_172/jre/bin/policytool
ADDED /java/linux-amd64/jdk1.8.0_172/jre/bin/rmid
ADDED /java/linux-amd64/jdk1.8.0_172/jre/bin/rmiregistry
ADDED /java/linux-amd64/jdk1.8.0_172/jre/bin/servertool
ADDED /java/linux-amd64/jdk1.8.0_172/jre/bin/tnameserv
ADDED /java/linux-amd64/jdk1.8.0_172/jre/bin/unpack200
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/jli/libjli.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/jvm.cfg
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libavplugin-53.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libavplugin-54.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libavplugin-55.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libavplugin-56.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libavplugin-57.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libavplugin-ffmpeg-56.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libavplugin-ffmpeg-57.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libawt.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libawt_headless.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libawt_xawt.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libbci.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libdcpr.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libdecora_sse.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libdeploy.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libdt_socket.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libfontmanager.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libfxplugins.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libglass.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libgstreamer-lite.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libhprof.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libinstrument.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libj2gss.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libj2pcsc.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libj2pkcs11.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libjaas_unix.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libjava.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libjava_crw_demo.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libjavafx_font.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libjavafx_font_freetype.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libjavafx_font_pango.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libjavafx_font_t2k.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libjavafx_iio.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libjawt.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libjdwp.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libjfr.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libjfxmedia.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libjfxwebkit.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libjpeg.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libjsdt.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libjsig.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libjsound.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libjsoundalsa.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libkcms.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/liblcms.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libmanagement.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libmlib_image.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libnet.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libnio.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libnpjp2.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libnpt.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libprism_common.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libprism_es2.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libprism_sw.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libresource.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libsctp.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libsplashscreen.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libsunec.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libt2k.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libunpack.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libverify.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/libzip.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/server/Xusage.txt
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/server/libjsig.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/amd64/server/libjvm.so
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/calendars.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/charsets.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/classlist
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/cmm/CIEXYZ.pf
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/cmm/GRAY.pf
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/cmm/LINEAR_RGB.pf
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/cmm/PYCC.pf
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/cmm/sRGB.pf
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/content-types.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/currency.data
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy/MixedCodeMainDialog.ui
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy/MixedCodeMainDialogJs.ui
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy/cautionshield.icns
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy/ffjcext.zip
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy/java-icon.ico
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy/messages.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy/messages_de.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy/messages_es.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy/messages_fr.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy/messages_it.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy/messages_ja.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy/messages_ko.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy/messages_pt_BR.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy/messages_sv.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy/messages_zh_CN.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy/messages_zh_HK.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy/messages_zh_TW.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy/mixcode_s.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy/splash.gif
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy/splash@2x.gif
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy/splash_11-lic.gif
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/deploy/splash_11@2x-lic.gif
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/applications/sun-java.desktop
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/applications/sun-javaws.desktop
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/applications/sun_java.desktop
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrast/16x16/apps/sun-java.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrast/16x16/apps/sun-javaws.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrast/16x16/apps/sun-jcontrol.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrast/16x16/mimetypes/gnome-mime-application-x-java-archive.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrast/16x16/mimetypes/gnome-mime-application-x-java-jnlp-file.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrast/16x16/mimetypes/gnome-mime-text-x-java.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrast/48x48/apps/sun-java.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrast/48x48/apps/sun-javaws.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrast/48x48/apps/sun-jcontrol.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrast/48x48/mimetypes/gnome-mime-application-x-java-archive.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrast/48x48/mimetypes/gnome-mime-application-x-java-jnlp-file.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrast/48x48/mimetypes/gnome-mime-text-x-java.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrastInverse/16x16/apps/sun-java.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrastInverse/16x16/apps/sun-javaws.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrastInverse/16x16/apps/sun-jcontrol.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrastInverse/16x16/mimetypes/gnome-mime-application-x-java-archive.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrastInverse/16x16/mimetypes/gnome-mime-application-x-java-jnlp-file.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrastInverse/16x16/mimetypes/gnome-mime-text-x-java.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrastInverse/48x48/apps/sun-java.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrastInverse/48x48/apps/sun-javaws.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrastInverse/48x48/apps/sun-jcontrol.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrastInverse/48x48/mimetypes/gnome-mime-application-x-java-archive.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrastInverse/48x48/mimetypes/gnome-mime-application-x-java-jnlp-file.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/HighContrastInverse/48x48/mimetypes/gnome-mime-text-x-java.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/LowContrast/16x16/apps/sun-java.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/LowContrast/16x16/apps/sun-javaws.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/LowContrast/16x16/apps/sun-jcontrol.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/LowContrast/16x16/mimetypes/gnome-mime-application-x-java-archive.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/LowContrast/16x16/mimetypes/gnome-mime-application-x-java-jnlp-file.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/LowContrast/16x16/mimetypes/gnome-mime-text-x-java.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/LowContrast/48x48/apps/sun-java.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/LowContrast/48x48/apps/sun-javaws.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/LowContrast/48x48/apps/sun-jcontrol.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/LowContrast/48x48/mimetypes/gnome-mime-application-x-java-archive.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/LowContrast/48x48/mimetypes/gnome-mime-application-x-java-jnlp-file.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/LowContrast/48x48/mimetypes/gnome-mime-text-x-java.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/hicolor/16x16/apps/sun-java.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/hicolor/16x16/apps/sun-javaws.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/hicolor/16x16/apps/sun-jcontrol.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/hicolor/16x16/mimetypes/gnome-mime-application-x-java-archive.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/hicolor/16x16/mimetypes/gnome-mime-application-x-java-jnlp-file.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/hicolor/16x16/mimetypes/gnome-mime-text-x-java.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/hicolor/48x48/apps/sun-java.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/hicolor/48x48/apps/sun-javaws.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/hicolor/48x48/apps/sun-jcontrol.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/hicolor/48x48/mimetypes/gnome-mime-application-x-java-archive.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/hicolor/48x48/mimetypes/gnome-mime-application-x-java-jnlp-file.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/icons/hicolor/48x48/mimetypes/gnome-mime-text-x-java.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/mime/packages/x-java-archive.xml
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/desktop/mime/packages/x-java-jnlp-file.xml
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/ext/cldrdata.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/ext/dnsns.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/ext/jaccess.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/ext/jfxrt.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/ext/localedata.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/ext/meta-index
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/ext/nashorn.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/ext/sunec.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/ext/sunjce_provider.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/ext/sunpkcs11.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/ext/zipfs.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/flavormap.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/fontconfig.RedHat.5.bfc
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/fontconfig.RedHat.5.properties.src
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/fontconfig.RedHat.6.bfc
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/fontconfig.RedHat.6.properties.src
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/fontconfig.SuSE.10.bfc
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/fontconfig.SuSE.10.properties.src
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/fontconfig.SuSE.11.bfc
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/fontconfig.SuSE.11.properties.src
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/fontconfig.Turbo.bfc
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/fontconfig.Turbo.properties.src
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/fontconfig.bfc
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/fontconfig.properties.src
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/fonts/LucidaBrightDemiBold.ttf
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/fonts/LucidaBrightDemiItalic.ttf
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/fonts/LucidaBrightItalic.ttf
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/fonts/LucidaBrightRegular.ttf
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/fonts/LucidaSansDemiBold.ttf
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/fonts/LucidaSansRegular.ttf
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/fonts/LucidaTypewriterBold.ttf
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/fonts/LucidaTypewriterRegular.ttf
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/fonts/fonts.dir
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/hijrah-config-umalqura.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/images/cursors/cursors.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/images/cursors/invalid32x32.gif
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/images/cursors/motif_CopyDrop32x32.gif
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/images/cursors/motif_CopyNoDrop32x32.gif
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/images/cursors/motif_LinkDrop32x32.gif
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/images/cursors/motif_LinkNoDrop32x32.gif
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/images/cursors/motif_MoveDrop32x32.gif
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/images/cursors/motif_MoveNoDrop32x32.gif
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/images/icons/sun-java.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/images/icons/sun-java_HighContrast.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/images/icons/sun-java_HighContrastInverse.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/images/icons/sun-java_LowContrast.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/javafx.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/javaws.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/jce.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/jexec
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/jfr.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/jfr/default.jfc
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/jfr/profile.jfc
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/jfxswt.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/jsse.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/jvm.hprof.txt
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/locale/de/LC_MESSAGES/sunw_java_plugin.mo
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/locale/es/LC_MESSAGES/sunw_java_plugin.mo
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/locale/fr/LC_MESSAGES/sunw_java_plugin.mo
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/locale/it/LC_MESSAGES/sunw_java_plugin.mo
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/locale/ja/LC_MESSAGES/sunw_java_plugin.mo
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/locale/ko.UTF-8/LC_MESSAGES/sunw_java_plugin.mo
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/locale/ko/LC_MESSAGES/sunw_java_plugin.mo
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/locale/pt_BR/LC_MESSAGES/sunw_java_plugin.mo
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/locale/sv/LC_MESSAGES/sunw_java_plugin.mo
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/locale/zh.GBK/LC_MESSAGES/sunw_java_plugin.mo
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/locale/zh/LC_MESSAGES/sunw_java_plugin.mo
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/locale/zh_HK.BIG5HK/LC_MESSAGES/sunw_java_plugin.mo
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/locale/zh_TW.BIG5/LC_MESSAGES/sunw_java_plugin.mo
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/locale/zh_TW/LC_MESSAGES/sunw_java_plugin.mo
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/logging.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/management-agent.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/management/jmxremote.access
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/management/jmxremote.password.template
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/management/management.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/management/snmp.acl.template
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/meta-index
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/net.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/oblique-fonts/LucidaSansDemiOblique.ttf
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/oblique-fonts/LucidaSansOblique.ttf
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/oblique-fonts/LucidaTypewriterBoldOblique.ttf
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/oblique-fonts/LucidaTypewriterOblique.ttf
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/oblique-fonts/fonts.dir
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/plugin.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/psfont.properties.ja
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/psfontj2d.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/resources.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/rt.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/security/blacklist
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/security/blacklisted.certs
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/security/cacerts
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/security/java.policy
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/security/java.security
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/security/javaws.policy
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/security/policy/limited/US_export_policy.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/security/policy/limited/local_policy.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/security/policy/unlimited/US_export_policy.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/security/policy/unlimited/local_policy.jar
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/security/trusted.libraries
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/sound.properties
ADDED /java/linux-amd64/jdk1.8.0_172/jre/lib/tzdb.dat
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja/man1/java.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja/man1/javaws.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja/man1/jjs.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja/man1/keytool.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja/man1/orbd.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja/man1/pack200.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja/man1/policytool.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja/man1/rmid.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja/man1/rmiregistry.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja/man1/servertool.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja/man1/tnameserv.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja/man1/unpack200.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja_JP.UTF-8/man1/java.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja_JP.UTF-8/man1/javaws.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja_JP.UTF-8/man1/jjs.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja_JP.UTF-8/man1/keytool.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja_JP.UTF-8/man1/orbd.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja_JP.UTF-8/man1/pack200.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja_JP.UTF-8/man1/policytool.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja_JP.UTF-8/man1/rmid.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja_JP.UTF-8/man1/rmiregistry.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja_JP.UTF-8/man1/servertool.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja_JP.UTF-8/man1/tnameserv.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/ja_JP.UTF-8/man1/unpack200.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/man1/java.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/man1/javaws.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/man1/jjs.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/man1/keytool.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/man1/orbd.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/man1/pack200.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/man1/policytool.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/man1/rmid.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/man1/rmiregistry.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/man1/servertool.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/man1/tnameserv.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/man/man1/unpack200.1
ADDED /java/linux-amd64/jdk1.8.0_172/jre/plugin/desktop/sun_java.desktop
ADDED /java/linux-amd64/jdk1.8.0_172/jre/plugin/desktop/sun_java.png
ADDED /java/linux-amd64/jdk1.8.0_172/jre/release
ADDED /lib/Java-8
ADDED /luts/16_colors.lut
ADDED /luts/5_ramps.lut
ADDED /luts/6_shades.lut
ADDED /luts/Cyan Hot.lut
ADDED /luts/Green Fire Blue.lut
ADDED /luts/HiLo.lut
ADDED /luts/ICA.lut
ADDED /luts/ICA2.lut
ADDED /luts/ICA3.lut
ADDED /luts/Magenta Hot.lut
ADDED /luts/Orange Hot.lut
ADDED /luts/Rainbow RGB.lut
ADDED /luts/Red Hot.lut
ADDED /luts/Thermal.lut
ADDED /luts/Yellow Hot.lut
ADDED /luts/blue_orange_icb.lut
ADDED /luts/brgbcmyw.lut
ADDED /luts/cool.lut
ADDED /luts/edges.lut
ADDED /luts/gem.lut
ADDED /luts/glasbey.lut
ADDED /luts/glasbey_inverted.lut
ADDED /luts/glasbey_on_dark.lut
ADDED /luts/glow.lut
ADDED /luts/mpl-inferno.lut
ADDED /luts/mpl-magma.lut
ADDED /luts/mpl-plasma.lut
ADDED /luts/mpl-viridis.lut
ADDED /luts/phase.lut
ADDED /luts/physics.lut
ADDED /luts/royal.lut
ADDED /luts/sepia.lut
ADDED /luts/smart.lut
ADDED /luts/thal.lut
ADDED /luts/thallium.lut
ADDED /luts/unionjack.lut
ADDED /macros/AutoRun/AutoRun_Scripts.ijm
ADDED /macros/StartupMacros.fiji.ijm
ADDED /macros/listManagement.txt
ADDED /macros/toolsets/Clear Custom Tools.txt
ADDED /macros/toolsets/Drawing Tools.txt
ADDED /macros/toolsets/Lookup Tables.txt
ADDED /plugins/Analyze/Dynamic_ROI_Profiler.clj
ADDED /plugins/Analyze/Measure_RGB.txt
ADDED /plugins/Examples/Add_Popup_Split_Channels.bsh
ADDED /plugins/Examples/Anaglyph_for_Red_Cyan_glasses.rb
ADDED /plugins/Examples/Blobs_Demo_in_Ruby.rb
ADDED /plugins/Examples/Command_Launchers/Command_Launcher_BeanShell.bsh
ADDED /plugins/Examples/Command_Launchers/Command_Launcher_Clojure.clj
ADDED /plugins/Examples/Command_Launchers/Command_Launcher_Javascript.js
ADDED /plugins/Examples/Command_Launchers/Command_Launcher_Python.py
ADDED /plugins/Examples/Command_Launchers/Command_Launcher_Ruby.rb
ADDED /plugins/Examples/CoverMaker/Cover_Maker.py
ADDED /plugins/Examples/CoverMaker/Prepare_Cover_Maker_Database.py
ADDED /plugins/Examples/Delayed_Snapshot.py
ADDED /plugins/Examples/Delayed_Snapshot_Window.bsh
ADDED /plugins/Examples/Edit_LUT_As_Text.py
ADDED /plugins/Examples/Extended_Profile_Plot.bsh
ADDED /plugins/Examples/Fiji_Cube.ijm
ADDED /plugins/Examples/Fiji_Logo_3D.js
ADDED /plugins/Examples/Find_Dimension_of_Raw_Image.py
ADDED /plugins/Examples/Image_To_Tool_Icon.bsh
ADDED /plugins/Examples/Multithreaded_Image_Processing.clj
ADDED /plugins/Examples/Multithreaded_Image_Processing_in_Javascript.js
ADDED /plugins/Examples/Plasma_Cloud.rb
ADDED /plugins/Examples/Same_Slice_in_Multiple_Images.rb
ADDED /plugins/Examples/The_Hue_Game.bsh
ADDED /plugins/Examples/TrakEM2_Example_Scripts/Homogenize_Ball_Radius.py
ADDED /plugins/Examples/TrakEM2_Example_Scripts/Measure_AreaLists.py
ADDED /plugins/Examples/TrakEM2_Example_Scripts/T2_Select_All.py
ADDED /plugins/Examples/TrakEM2_Example_Scripts/T2_set_all_transforms_to_identity.py
ADDED /plugins/Examples/TrakEM2_Example_Scripts/extract_stack_under_arealist.py
ADDED /plugins/Examples/blend_two_images.clj
ADDED /plugins/Examples/celsius_to_fahrenheit.clj
ADDED /plugins/Examples/chess_.py
ADDED /plugins/Examples/compose_rgb-stacks.bsh
ADDED /plugins/Examples/downsample_.js
ADDED /plugins/Examples/list_all_threads.py
ADDED /plugins/Examples/random_noise_example.clj
ADDED /plugins/JRuby/imagej.rb
ADDED /plugins/Macros/About_Plugin_Macros.txt
ADDED /plugins/Macros/Bulls_Eye.txt
ADDED /plugins/Macros/Polygon_.txt
ADDED /plugins/Macros/RGB_Histogram.txt
ADDED /plugins/Macros/batch_convert_any_to_tif.txt
ADDED /plugins/Scripts/File/Fix_Funny_Filenames.ijm
ADDED /plugins/Scripts/File/Make_Screencast.bsh
ADDED /plugins/Scripts/File/Open_Samples/Adelsons_Squares.ijm
ADDED /plugins/Scripts/File/Open_Samples/Comparing_Lengths.ijm
ADDED /plugins/Scripts/File/Open_Samples/Spirals_.ijm
ADDED /plugins/Scripts/File/Open_Samples/Straight_Lines.ijm
ADDED /plugins/Scripts/File/Open_Samples/Striped_Circles.ijm
ADDED /plugins/Scripts/Image/Adjust/Scale_to_DPI.js
ADDED /plugins/Scripts/Image/Color/Set_Color_By_Wavelength.ijm
ADDED /plugins/Scripts/Image/Hyperstacks/Temporal-Color_Code.ijm
ADDED /plugins/Scripts/Plugins/AutoRun/Check_Required_Update_Sites.js
ADDED /plugins/Scripts/Plugins/Utilities/Create_Desktop_Icon.bsh
ADDED /plugins/Scripts/Plugins/Utilities/Find_Jar_For_Class.bsh
ADDED /plugins/Scripts/Plugins/Utilities/Record_Desktop.py
ADDED /plugins/Scripts/Plugins/Utilities/Record_Window.py
ADDED /plugins/Utilities/Close_All_Without_Saving.txt
ADDED /plugins/readme.txt
ADDED /retro/backport-util-concurrent-3.1.jar
ADDED /retro/retrotranslator-runtime-1.2.9.jar
ADDED /retro/retrotranslator-transformer-1.2.9.jar
ADDED /scripts/ImageJ.m
ADDED /scripts/InstallJava3D.m
ADDED /scripts/IsJava3DInstalled.m
ADDED /scripts/Matlab3DViewerDemo_1.m
ADDED /scripts/Matlab3DViewerDemo_2.m
ADDED /scripts/Matlab3DViewerDemo_3.m
ADDED /scripts/Matlab3DViewerIntroduction.m
ADDED /scripts/Miji.m
ADDED /scripts/Miji_Test.m
ADDED /scripts/bfopen.m
ADDED /scripts/copytoImg.m
ADDED /scripts/copytoImgPlus.m
ADDED /scripts/copytoMatlab.m
ADDED /scripts/importTrackMateTracks.m
ADDED /scripts/trackmateEdges.m
ADDED /scripts/trackmateFeatureDeclarations.m
ADDED /scripts/trackmateGraph.m
ADDED /scripts/trackmateImageCalibration.m
ADDED /scripts/trackmateSpots.m
DELETED /jars/VectorGraphics2D-0.13.jar
DELETED /jars/c3p0-0.9.5.3.jar
DELETED /jars/cdm-4.6.13.jar
DELETED /jars/commons-text-1.6.jar
DELETED /jars/ffmpeg-4.1-1.4.4-android-arm.jar
DELETED /jars/ffmpeg-4.1-1.4.4-android-arm64.jar
DELETED /jars/ffmpeg-4.1-1.4.4-android-x86.jar
DELETED /jars/ffmpeg-4.1-1.4.4-android-x86_64.jar
DELETED /jars/ffmpeg-4.1-1.4.4-linux-armhf.jar
DELETED /jars/ffmpeg-4.1-1.4.4-linux-ppc64le.jar
DELETED /jars/ffmpeg-4.1-1.4.4-linux-x86.jar
DELETED /jars/ffmpeg-4.1-1.4.4-macosx-x86_64.jar
DELETED /jars/ffmpeg-4.1-1.4.4-windows-x86.jar
DELETED /jars/ffmpeg-4.1-1.4.4-windows-x86_64.jar
DELETED /jars/google-http-client-1.20.0.jar
DELETED /jars/google-http-client-xml-1.20.0.jar
DELETED /jars/httpmime-4.5.1.jar
DELETED /jars/httpservices-4.6.13.jar
DELETED /jars/jackson-annotations-2.9.8.jar
DELETED /jars/jackson-core-2.9.8.jar
DELETED /jars/jackson-databind-2.9.8.jar
DELETED /jars/jfreesvg-3.4.jar
DELETED /jars/mchange-commons-java-0.2.15.jar
DELETED /jars/minio-5.0.2.jar
DELETED /jars/okhttp-3.6.0.jar
DELETED /jars/okio-1.11.0.jar
DELETED /jars/ome-codecs-0.2.5.jar
DELETED /jars/scijava-io-http-0.2.0.jar
DELETED /jars/scijava-plugins-io-table-0.2.0.jar
DELETED /jars/xchart-3.5.4.jar
DELETED /jars/xpp3-1.1.4c.jar
MODIFED /jars/jep-2.4.2.jar -> /jars/jep-2.4.2.jar
VERSION_CHANGED /jars/T2-NIT (1.1.3 -> 1.1.2)
VERSION_CHANGED /jars/T2-TreelineGraph (1.1.3 -> 1.1.2)
VERSION_CHANGED /jars/autocomplete (3.0.0 -> 2.6.1)
VERSION_CHANGED /jars/batik-anim (1.11 -> 1.10)
VERSION_CHANGED /jars/batik-awt-util (1.11 -> 1.10)
VERSION_CHANGED /jars/batik-bridge (1.11 -> 1.10)
VERSION_CHANGED /jars/batik-constants (1.11 -> 1.10)
VERSION_CHANGED /jars/batik-css (1.11 -> 1.10)
VERSION_CHANGED /jars/batik-dom (1.11 -> 1.10)
VERSION_CHANGED /jars/batik-ext (1.11 -> 1.10)
VERSION_CHANGED /jars/batik-gvt (1.11 -> 1.10)
VERSION_CHANGED /jars/batik-i18n (1.11 -> 1.10)
VERSION_CHANGED /jars/batik-parser (1.11 -> 1.10)
VERSION_CHANGED /jars/batik-script (1.11 -> 1.10)
VERSION_CHANGED /jars/batik-svg-dom (1.11 -> 1.10)
VERSION_CHANGED /jars/batik-svggen (1.11 -> 1.10)
VERSION_CHANGED /jars/batik-util (1.11 -> 1.10)
VERSION_CHANGED /jars/batik-xml (1.11 -> 1.10)
VERSION_CHANGED /jars/bio-formats/formats-api (6.3.0 -> 6.4.0)
VERSION_CHANGED /jars/bio-formats/formats-bsd (6.3.0 -> 6.4.0)
VERSION_CHANGED /jars/bio-formats/formats-gpl (6.3.0 -> 6.4.0)
VERSION_CHANGED /jars/bio-formats/jxrlib-all (0.2.1 -> 0.2.2)
VERSION_CHANGED /jars/bio-formats/turbojpeg (6.3.0 -> 6.4.0)
VERSION_CHANGED /jars/commons-codec (1.12 -> 1.11)
VERSION_CHANGED /jars/commons-compress (1.18 -> 1.4.1)
VERSION_CHANGED /jars/commons-lang3 (3.9 -> 3.8.1)
VERSION_CHANGED /jars/fiji (2.0.0-SNAPSHOT -> 2.0.0-pre-8)
VERSION_CHANGED /jars/gluegen-rt (2.3.2-natives-android-aarch64 -> 2.3.2)
VERSION_CHANGED /jars/httpclient (4.5.1 -> 4.5.6)
VERSION_CHANGED /jars/ij (1.52r -> 1.52p)
VERSION_CHANGED /jars/ij1-patcher (1.0.0 -> 0.12.9)
VERSION_CHANGED /jars/imagej (2.0.0-rc-72 -> 2.0.0-rc-69)
VERSION_CHANGED /jars/imagej-common (0.31.0 -> 0.28.2)
VERSION_CHANGED /jars/imagej-legacy (0.35.0 -> 0.35.2)
VERSION_CHANGED /jars/imagej-notebook (0.7.1 -> 0.3.0)
VERSION_CHANGED /jars/imagej-ops (0.45.2 -> 0.43.1)
VERSION_CHANGED /jars/imagej-plugins-uploader-webdav (0.2.2 -> 0.3.1)
VERSION_CHANGED /jars/imagej-ui-swing (0.22.0 -> 0.23.1)
VERSION_CHANGED /jars/imagej-updater (0.9.3 -> 0.10.4)
VERSION_CHANGED /jars/imglib2 (5.8.0 -> 5.6.3)
VERSION_CHANGED /jars/imglib2-roi (0.8.1 -> 0.7.0)
VERSION_CHANGED /jars/itextpdf (5.5.13.1 -> 5.5.13)
VERSION_CHANGED /jars/javassist (3.25.0-GA -> 3.24.0-GA)
VERSION_CHANGED /jars/jgrapht-core (1.3.1 -> 1.3.0)
VERSION_CHANGED /jars/jheaps (0.10 -> 0.9)
VERSION_CHANGED /jars/jitk-tps (3.0.1 -> 3.0.0)
VERSION_CHANGED /jars/joal (2.3.2-natives-android-aarch64 -> 2.3.2)
VERSION_CHANGED /jars/jocl (2.3.2-natives-android-aarch64 -> 2.3.2)
VERSION_CHANGED /jars/joda-time (2.10.2 -> 2.10.1)
VERSION_CHANGED /jars/jogl-all (2.3.2-natives-android-aarch64 -> 2.3.2)
VERSION_CHANGED /jars/joml (1.9.16 -> 1.9.12)
VERSION_CHANGED /jars/jython-shaded (2.7.1.1 -> 2.7.1)
VERSION_CHANGED /jars/languagesupport (3.0.0 -> 2.6.0)
VERSION_CHANGED /jars/metadata-extractor (2.11.0 -> 2.6.2)
VERSION_CHANGED /jars/minimaven (2.2.2 -> 2.2.1)
VERSION_CHANGED /jars/native-lib-loader (2.3.4 -> 2.3.2)
VERSION_CHANGED /jars/perf4j (0.9.16 -> 0.9.13)
VERSION_CHANGED /jars/rsyntaxtextarea (3.0.3 -> 2.6.1)
VERSION_CHANGED /jars/scijava-common (2.82.0 -> 2.80.1)
VERSION_CHANGED /jars/scijava-search (0.6.0 -> 0.5.0)
VERSION_CHANGED /jars/scijava-table (0.4.0 -> 0.2.0)
VERSION_CHANGED /jars/scijava-ui-swing (0.13.0 -> 0.12.0)
VERSION_CHANGED /jars/script-editor (0.5.0 -> 0.5.1)
VERSION_CHANGED /jars/slf4j-api (1.7.26 -> 1.7.25)
VERSION_CHANGED /jars/snakeyaml (1.24 -> 1.23)
VERSION_CHANGED /jars/xmpcore (5.1.3 -> 5.1.2)
VERSION_CHANGED /plugins/AnalyzeSkeleton_ (3.4.1 -> 3.3.0)
VERSION_CHANGED /plugins/Stack_Manipulation (2.1.1 -> 2.1.2)
VERSION_CHANGED /plugins/TrackMate_ (4.0.0 -> 5.2.0)
VERSION_CHANGED /plugins/Trainable_Segmentation (3.2.29 -> 3.2.34)
VERSION_CHANGED /plugins/Vaa3d_Writer (1.0.3 -> 1.0.2)
VERSION_CHANGED /plugins/bUnwarpJ_ (2.6.11 -> 2.6.12)
VERSION_CHANGED /plugins/bio-formats_plugins (6.3.0 -> 6.4.0)
VERSION_CHANGED /plugins/register_virtual_stack_slices (3.0.5 -> 3.0.7)
MOVED /jars/Correct_3D_Drift-1.0.3.jar -> /plugins/Correct_3D_Drift-1.0.3.jar
MOVED /jars/KymographBuilder-2.1.1.jar -> /plugins/KymographBuilder-2.1.1.jar
MOVED /jars/bigdataviewer_fiji-6.0.0.jar -> /plugins/bigdataviewer_fiji-6.0.0.jar
MOVED /jars/ffmpeg-4.1-1.4.4-linux-x86_64.jar -> /jars/linux64/ffmpeg-4.1-1.4.4-linux-x86_64.jar
MOVED /jars/metakit-5.3.2.jar -> /jars/bio-formats/metakit-5.3.2.jar
MOVED /jars/ome-common-6.0.4.jar -> /jars/bio-formats/ome-common-6.0.4.jar
MOVED /jars/ome-jai-0.1.0.jar -> /jars/bio-formats/ome-jai-0.1.0.jar
MOVED /jars/ome-mdbtools-5.3.2.jar -> /jars/bio-formats/ome-mdbtools-5.3.2.jar
MOVED /jars/ome-poi-5.3.3.jar -> /jars/bio-formats/ome-poi-5.3.3.jar
MOVED /jars/ome-xml-6.0.1.jar -> /jars/bio-formats/ome-xml-6.0.1.jar
MOVED /jars/specification-6.0.1.jar -> /jars/bio-formats/specification-6.0.1.jar
```

</p>
</details>

I added the class to this branch for now and will track my progress here if I make any, I don't think this branch will necessarily ever be merged, but it might produce separate PRs fixing individual issues.

ctrueden commented:
> I tried a couple of command line comparison tools but nothing was super helpful or colored.

How about this:
```
(cd Fiji.app && find .) | sort > app.txt
(cd Fiji.build && find .) | sort > build.txt
git diff --no-index --word-diff --word-diff-regex='[^/-]+' app.txt build.txt
```

ctrueden commented:
For what it's worth—as I am working on improving this functionality this week—I pushed a script for diffing directories: ctrueden/ctr-scripts@452a6b5d7c90d9243af9497af7af495b62e17320. Useful for comparing two Fiji installations.

ctrueden commented:
With scijava-maven-plugin 2.0.0, the `copy-jars` was renamed to `populate-app`. This renamed goal is now intended for either modifying an existing SciJava application (e.g. ImageJ or Fiji) environment, or building one from scratch, although it currently still falls short of this intent. But at least the naming is sensible now, and the bugs relating to native classifier JARs have been squashed!

ctrueden commented:
I'm going to close this PR, since the `populate-app` exists and is the direction we want to take this. We may still want to add a feature where it downloads a JDK as part of the environment construction... but maybe not. Will revisit this later as needs arise.
